librz/signature/pac.c

```diff
@@ -0,0 +1,369 @@
+// SPDX-FileCopyrightText: 2021 RizinOrg <info@rizin.re>
+// SPDX-FileCopyrightText: 2021 deroad <wargio@libero.it>
+// SPDX-License-Identifier: LGPL-3.0-only
```

Comment:
Can you please add a short explanation of what pac is and a rough file format structure if any? Using Doxygen.